worker/src/main/java/gov/cms/ab2d/worker/service/BeneficiaryServiceImpl.java

```diff
@@ -0,0 +1,85 @@
+package gov.cms.ab2d.worker.service;
+
+import gov.cms.ab2d.common.model.Beneficiary;
+import gov.cms.ab2d.common.model.Contract;
+import gov.cms.ab2d.common.model.Coverage;
+import gov.cms.ab2d.common.repository.BeneficiaryRepository;
+import gov.cms.ab2d.common.repository.ContractRepository;
+import gov.cms.ab2d.common.repository.CoverageRepository;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Propagation;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.LinkedHashSet;
+import java.util.Optional;
+import java.util.Set;
+
+@Slf4j
+@Service
+@RequiredArgsConstructor
+public class BeneficiaryServiceImpl implements BeneficiaryService {
+
+    private final BeneficiaryRepository beneRepo;
+    private final ContractRepository contractRepo;
+    private final CoverageRepository coverageRepo;
+
+
+    /**
+     * Given a contractId and a month,
+     * search for bene information in the local db
+     *
+     * @param contractId
+     * @param month
+     * @return
+     */
+    @Override
+    public Set<String> findPatientIdsInDb(Long contractId, int month) {
+        var patientIds = coverageRepo.findActivePatientIds(contractId, month);
+        return new LinkedHashSet<>(patientIds);
```

Comment:
Since this is being ordered, should there be an order by statement in the SQL query?

Reply:
And is it possible for the JPA method to return a linkedhashset?

Reply:
I do not really think we need to do an order by since the records are written in a transaction.
I also did a quick search on having Spring data return a Set instead of a List by default and it does not seem like there is an easy/simple way. 
The above code to use the constructor is clear & simple and I am inclined to leave it as is.

Reply:
Ok. I'm a little confused by why it's keeping order with the patients in the first place then with the linkedhashset since the db wouldn't return order in a predictable way?

Reply:
Do I need to do an order by @dkrylovsb ?  
If yes, which column should I use? 
It does not seem like `contract_id` or `beneficiary_id` would make sense to do an `order by`.

Reply:
DB orders by primary key by default; this should work for us

Reply:
Ok, didn't realize it would order by the PK by default.